<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calcClass</class>
 <widget class="QMainWindow" name="calcClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>calc</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(217, 229, 240)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QPushButton" name="result_btn">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(199, 214, 255)</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="multiply_btn">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(199, 214, 255)</string>
        </property>
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="factorial_btn">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(199, 214, 255)</string>
        </property>
        <property name="text">
         <string>!</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="divide_btn">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(199, 214, 255)</string>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="power_btn">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(199, 214, 255)</string>
        </property>
        <property name="text">
         <string>^</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="minus_btn">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(199, 214, 255)</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="plus_btn">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(199, 214, 255)</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="sqrt_btn">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(199, 214, 255)</string>
        </property>
        <property name="text">
         <string>SQRT</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="clear_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(240, 240, 240)</string>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(240, 240, 240)</string>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QPushButton" name="num_2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220, 220, 220)</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="num_1">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220, 220, 220)</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="num_5">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220, 220, 220)</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="sign_btn">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220, 220, 200)</string>
        </property>
        <property name="text">
         <string>+/-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="num_3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220, 220, 220)</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="num_0">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220, 220, 220)</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="num_7">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220, 220, 220)</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="num_6">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220, 220, 220)</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="num_8">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220, 220, 220)</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="num_9">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220, 220, 220)</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="point_btn">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220, 220, 200)</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="num_4">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(220, 220, 220)</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="e_btn">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 170, 127)</string>
        </property>
        <property name="text">
         <string>e</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pi_btn">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 170, 127)</string>
        </property>
        <property name="text">
         <string>pi</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="g_btn">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 170, 127)</string>
        </property>
        <property name="text">
         <string>g</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" rowspan="2" colspan="2">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>380</width>
        <height>55</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(205, 227, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="history_screen">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="screen">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>10</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Verdana</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="calc.qrc"/>
 </resources>
 <connections/>
</ui>
